misc/debug-adapter/modules/debug-adapter-core/src/main/java/org/ballerinalang/debugadapter/evaluation/engine/expression/MethodCallExpressionEvaluator.java

```diff
@@ -248,14 +249,20 @@ && modifyName(symbol.getName().get()).equals(className))
                 .findFirst()
                 .map(symbol -> (ClassSymbol) symbol);
     }
-
-    protected Optional<MethodSymbol> findObjectMethodInClass(ClassSymbol classDef, String methodName) {
+    protected Optional<MethodSymbol> findObjectMethodInClass(ClassSymbol classDef,
+                                                             String methodName,
+                                                             Qualifier qualifier) {
```

Comment:
Shouldn't we accept a list/collection of qualifiers instead, as there can be more than one qualifier for a given method?

Reply:
You accept list of qualifiers and we select the function which has all these qualifiers?

Reply:
yes exactly. So in your usecase, we can pass a singleton list right?

Reply:
Sure